woocommerce/woocommerce
# CYS: hide popover when the mouse pointer leaves the site preview and then back.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #48182 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you have the last version of Gutenberg enabled
2. Make sure you have the WooCommerce beta tester plugin installed
3. Head over to Tools > WCA Test Helper > Features
4. Make sure you see the `pattern-toolkit-full-composability` on the list.
5. Enable it.
6. Head over to WooCommerce > Home > Customize your store.
7. On the main iframe, click on a block.
8. Ensure that a tooltip appears.
9. Move the mouse pointer and ensure that the tooltip follows the pointer.
10. Hover another pattern.
11. Ensure that the tooltip disappears.
12. Click another pattern.
13. Ensure that the tooltip position changes.
14. Move the pointer on the left sidebar.
15. Ensure that the tooltip disappears.
16. Move the pointer on the iframe again.
17. Ensure that the tooltip doesn't appear.
18. Click on a block
19. Ensure that the tooltip appears.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: hide popover when the mouse pointer leaves the site preview and then back.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
